nixos/modules/services/security/cfssl.nix

```diff
@@ -164,6 +164,12 @@ in
       ];
       description = "Log level (0 = DEBUG, 5 = FATAL).";
     };
+
+    disable = lib.mkOption {
+      default = null;
+      type = lib.types.nullOr lib.types.str;
+      description = "Endpoints to disable (comma-separated list)";
```

Comment:
nit: there's `types.commas` a string where multiple definitions are concatenated with a comma in between.